easybuild/tools/config.py

```diff
@@ -516,15 +378,14 @@ def log_path():
 
 def get_build_log_path():
     """
-    return temporary log directory
+    Return (temporary) directory for build log
     """
     variables = ConfigurationVariables()
-    if 'tmp_logdir' in variables:
-        return variables['tmp_logdir']
+    if variables['tmp_logdir'] is not None:
+        res = variables['tmp_logdir']
     else:
-        # TODO remove default setting. it should have been set through options
-        _log.deprecated('tmp_logdir not set in config, returning default: %s' % DEFAULT_TMP_LOGDIR, "2.0")
-        return DEFAULT_TMP_LOGDIR
+        res = tempfile.gettempdir()
+    return res
```

Comment:
this is not fully equivalent... what is `tmp_logdir` is set to '' (which would be stupid, but fine)